components/signersForm.tsx

```diff
@@ -191,12 +192,17 @@ const SignersForm: FC<{
         if (values.requiredSignatures > values.validators.length) {
           errors.requiredSignatures = `threshold too high. required number of signatures: ${values.requiredSignatures}, total amount of signers: ${values.validators.length}`;
         }
+
+        if (isNaN(parseInt(values.effectivePeriod as any))) {
```

Comment:
It seems to be a JS thing that `parseInt("12a0test")` will be `12` ?
Maybe use another way to check number? 
(I personally recommend not use `any` if possible)